internal/pkg/n8n-client-go/workflows.go

```diff
@@ -93,3 +93,26 @@ func (c *Client) DeleteWorkflow(workflowID string) (*Workflow, error) {
 
 	return &workflow, nil
 }
+
+// ActivateWorkflow - Activates a workflow by its ID.
+func (c *Client) ActivateWorkflow(workflowID string) (*Workflow, error) {
+	url := fmt.Sprintf("%s/api/v1/workflows/%s/activate", c.HostURL, workflowID)
+
+	req, err := http.NewRequest("POST", url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	body, err := c.doRequest(req, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var workflow Workflow
```

Comment:
Same point as above.
```suggestion
	workflow := Workflow{}
```